### ISSUE TITLE
Missing README file in the published package csv-load-sync

### ISSUE THREAD

**Issue by bahmutov** [npm/npm#11436]
Hi,

The git repo for my package csv-load-sync has README.md, but the published package is missing it. I looked at all previous NPM issues for the missing README file and none applies. For example, the list of packaged files includes README.md

```
> csv-load-sync@0.0.0-semantic-release size /Users/gleb/git/csv-load-sync
> t="$(npm pack .)"; wc -c "${t}"; tar tvf "${t}"; rm "${t}";

    2412 csv-load-sync-0.0.0-semantic-release.tgz
-rw-r--r--  0 501    20        801 Feb  5 09:40 package/package.json
-rw-r--r--  0 501    20       2598 Jan 29 12:22 package/README.md
-rw-r--r--  0 501    20         39 Dec 29  2014 package/index.js
-rw-r--r--  0 501    20       1835 Feb  1 09:14 package/src/load.js
```

I see `readmeFilename` blank

```
$ npm info csv-load-sync
{
  license: 'MIT',
  readmeFilename: '',
}
```

Links:

* https://www.npmjs.com/package/csv-load-sync
* repo https://github.com/bahmutov/csv-load-sync

**Comment by ashleygwilliams** [comment]
hi! i test published your project using the exact `README` and `package.json` using `node 4.2.1` and `npm 3.7.1`. You can see it here: https://www.npmjs.com/package/issues-readme-test

`npm info` also shows the readmeFileName. what version of npm/node are you using?

**Comment by ashleygwilliams** [comment]
we experienced some issues with publishing yesterday (when it appears you published). assuming that is what caused this error, you should attempt to publish again. lemme know how it goes!

**Comment by bahmutov** [comment]
Weird, I unpublished, published again, `node 5.1.0` and `npm 3.7.2` - `npm info csv-load-sync` shows the readme filename, but the website still says not found. Maybe I should just give it time. This is not very important, the github repo is there, so people can find it.
